docs/changelog.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+## CARET
+
+### v0.2.3 <small>\_ July 14, 2022</small> {id = "2.3.0"}
+
+- Supported OS: Ubuntu 20.04
+- Supported ROS Dist.: ROS Galactic
+- Improved output message from `path.verify()` method
+- Improved view of graphs
+- Added wildcard support for `Architecture.callbacks()` method
+- Added command line function to check trace data
+- Fixed specification of `Architecture.search_paths()` method
+- Added function to extract duplicated timer callback
+
+### v0.2.3 <small>\_ July 14, 2022</small> {id = "2.2.0"}
```

Comment:
Is this a mistake for v0.2.2?

Reply:
@hsgwa 
resolved at https://github.com/tier4/CARET_doc/pull/69/commits/751ce41bd38ee9ed3392d361cf0c4c328088a924